### ISSUE TITLE
extend components to display a loading gif

### ISSUE THREAD

**Issue by lbelinsk** [wix/wix-style-react#59]
I have a use case where I need to display a 'loading' icon above the Input field.

I have found kind of a backdoor workaround to do that with the **Loader** component and the 'unit' props:

`const loader = <Loader size="small"/>;`
`<Input ... unit={loader}/>`

Here is a snapshot (ignore the 'LOADING' placeholder...):

![screen shot 2016-12-17 at 9 40 08 pm](https://cloud.githubusercontent.com/assets/18248989/21289278/6bccc702-c4a1-11e6-9477-5109cfdd6ac1.png)

since the props validation of Input expects unit of type string, this triggers an error, as expected..
I thought maybe it is worth to provide a nicer option to do that, maybe not for Input, but for components with data list suggestions which might be fetched from a server asynchronously, like it did in my use case.

**Comment by lbelinsk** [comment]
Also, it might be useful to add a 'disabled' version of the Input field (kind of grey background) during the data loading...

**Comment by mastertheblaster** [comment]
@lbelinsk Currently the input is smart enough and exposes `prefix` and `suffix` properties.

```js
<Input prefix={<Loader size="small"/>} suffix={<Loader size="small"/>}/>
```

Though I do agree there is no disabled option yet.

<img width="439" alt="screen shot 2017-01-19 at 10 06 41" src="https://cloud.githubusercontent.com/assets/10008149/22098440/0146a4d2-de2f-11e6-8284-a05213c57481.png">
<img width="218" alt="screen shot 2017-01-19 at 10 06 35" src="https://cloud.githubusercontent.com/assets/10008149/22098441/014a4164-de2f-11e6-9323-8e6102050022.png">

**Comment by lbelinsk** [comment]
We have a disabled option as well, so I am closing this one..
